arwen/host/breakpoints.go

```diff
@@ -10,13 +10,13 @@ func (host *vmHost) handleBreakpointIfAny(executionErr error) error {
 	}
 
 	runtime := host.Runtime()
+	runtime.AddError(executionErr)
```

Comment:
I think the code before was safer. Like we do not need to add error in case of asyncCall breakpoint. and execution error is not important than.

Reply:
Agree about `executionErr`, will try to find a way to make certain tests pass without this line.

But are we sure we do not want the error returned by `handleAsyncCallBreakpoint()` below? Previously it was ignored, but now it is added.

Reply:
The second part of the changes are good. but there is no need to AddError(executionErr) in case of breakpoiintAsyncCall

Reply:
Removed.